srcpkgs/droidcam/template

```diff
@@ -0,0 +1,31 @@
+# Template file for 'droidcam'
+pkgname=droidcam
+version=1.7.1
+revision=1
+makedepends="pkg-config libjpeg-turbo-devel ffmpeg-devel alsa-lib-devel speex-devel libusbmuxd-devel libplist-devel gtk+3-devel"
+depends="v4l2loopback"
+short_desc="Tool for using your android device as a wireless/usb webcam"
+maintainer="ibrokemypie <ibrokemypie@bastardi.net>"
+license="GPL-2.0-or-later"
+homepage="https://github.com/dev47apps/droidcam"
+distfiles="https://github.com/dev47apps/${pkgname}/archive/v${version}.tar.gz"
+checksum="1f925c093df165afcd39e76b12a3db0f19fabbf15a11e1c7b55a97bba331f113"
+nopie=true
+
+do_build() {
+	# All JPEG* parameters are needed to use shared version of libturbojpeg instead of
+	# static one.
+	#
+	# Also libusbmuxd requires an override while linking.
+	make JPEG_DIR="" JPEG_INCLUDE="" JPEG_LIB="" JPEG=$(pkg-config --libs --cflags libturbojpeg) USBMUXD=-lusbmuxd-2.0
+
+}
+
+do_install() {
+	vbin ${PKGNAME}
+	vbin ${PKGNAME}-cli
+	vinstall icon2.png 644 /usr/share/pixmaps/ ${PKGNAME}.png
+	vinstall icon2.png 644 /opt/ ${PKGNAME}-icon.png
```

Comment:
This shouldn't touch `/opt`... I don't see anything that would even use the file. And if it really goes in `/opt`, it should be inside its own directory.

And in this case you can use `droidcam` (preferred) or `${pkgname}`, not `${PKGNAME}`.